airbyte-integrations/connectors/source-declarative-manifest/unit_tests/test_source_declarative_local_manifest.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import patch
+
+import pytest
+from source_declarative_manifest import run
+
+POKEAPI_JSON_SPEC_SUBSTRING = '"required":["pokemon_name"]'
+SUCCESS_CHECK_SUBSTRING = '"connectionStatus":{"status":"SUCCEEDED"}'
+
+@pytest.fixture(autouse=True)
+def setup(valid_local_manifest_yaml):
+    with patch('source_declarative_manifest.run._is_local_manifest_command', return_value=True):
+        with patch('source_declarative_manifest.run.YamlDeclarativeSource._read_and_parse_yaml_file', return_value=valid_local_manifest_yaml):
```

Comment:
@bnchrch Should we also have a test where the return value is an _invalid_ YAML manifest? 
Or does this code path not check that? And if so, should we?

Reply:
Good idea!